Microsoft.JSInterop/Microsoft.JSInterop.Infrastructure/DotNetDispatcher.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+
+namespace Microsoft.JSInterop.Infrastructure
+{
+	/// <summary>
+	/// Provides methods that receive incoming calls from JS to .NET.
+	/// </summary>
+	public static class DotNetDispatcher
+	{
+		private readonly struct AssemblyKey : IEquatable<AssemblyKey>
+		{
+			public Assembly? Assembly
+			{
+				[System.Runtime.CompilerServices.NullableContext(2)]
+				[CompilerGenerated]
+				get
+				{
+					throw null;
+				}
+			}
+
+			public string AssemblyName
```

Comment:
Shouldn't this be nullable as well? If `Assembly` is nullable, then so should `AssemblyName`.

Reply:
This one works fine. Either an Assembly or an AssemblyName is always passed in. When the former is passed in, we use it to calculate the assembly name